README.md

```diff
@@ -1 +1,248 @@
+
+![](cognitive_burden.png)
+
+## Features
+
+## Other considerations
+
+__Assumption:__ We operate on open-world assumptions (OWA), not closed-world assumptions (CSA). In OWA, we do not make any assumptions about the absence of statements, while in CSA absence of statements would be evaluated as false, i.e., assumed to be false.
+
+## Architecture
+The figure below shows the high-level overview of the components of the BrainKB architecture.
+
+![](ingest.png)
+_Figure 2: Ingest Service_
+
+__Application:__ The application (or the application layer) is the go-to point that provides access to BrainKB, such as via UI.
+
+__Service:__ The service layer implements the core logic and is broken down into multiple services based on the functionalities.  
+__Resource:__ The resource will provide the necessary computational resources that are required to deliver the required service by BrainKB.
+
+![](initial-arch.png)
+_Figure 3: Architecture of BrainKB_
+
+## Target Audience
+
+- **Neuroscience researchers:** BrainKB's primary audience is neuroscience researchers, who can use the platform to integrate, visualize, and analyze neuroscience data. They can also capitalize on the platform's ability to synthesize their data (or knowledge) into KGs.
+
+- **Research Labs and Academic Institutions:** BrainKB will be an invaluable resource for teaching and research in academic contexts specializing in neuroscience research. It offers convenient access to integrated neuroscience data for faculty and students.
+
+- **Policy Makers and Healthcare Professionals:** Neurology policymakers can use BrainKB, aka the neuroscience knowledge that BrainKB hosts, to make policy decisions. Furthermore, healthcare professionals in neurology (or clinical neuroscience) may also use BrainKB knowledge to understand and improve neurological disease outcomes.
+
+- **Neuroscience-related Companies:** Companies specializing in developing drugs for neurological diseases can use the platform's KGs to gain insights into neurological conditions and treatments.
+
+## Use cases
+
+- **Extraction/Integration/Refinement:** BrainKB will provide features to extract knowledge from diverse sources, such as raw text and scientific publications, and integrate it with the knowledge represented via KGs. Additionally, BrainKB also provides features to refine the extracted knowledge, e.g., through humans in the loop.
+
+- **Cards:** The BrainKB web application allows easy visualization of the knowledge of interest to scientists/researchers stored in KGs and their corresponding interconnected knowledge. The figure below (Figure 4) shows a snippet of the entity card from the BrainKB web application, which can be accessed at [http://beta.brainkb.org](http://beta.brainkb.org).
```

Comment:
```suggestion
- **Cards:** The BrainKB web application allows easy visualization of the knowledge of interest to scientists/researchers stored in KGs and their corresponding interconnected knowledge. Figure 4 shows a snippet of the entity card from the BrainKB web application, which can be accessed at [http://beta.brainkb.org](http://beta.brainkb.org).
```
The first sentence is pretty complex, but not sure how to change it. Is this figure really about "knowledge" or just data/information?